src/shared/components/palette/email/write-email-field.html

```diff
@@ -1,11 +1,12 @@
 <div class="form-group bottom-30" [ngClass]="{'form-group-error': !emailControl.valid && emailControl.touched}">
 
   <label [for]="id()">
-    <span class="form-label" *ngIf="caseField.label">{{caseField | ccdFieldLabel}}</span>
-    <span class="form-hint" *ngIf="caseField.hint_text">{{caseField.hint_text}}</span>
-    <span class="error-message" *ngIf="emailControl.errors && emailControl.touched">{{emailControl.errors | ccdFirstError}}</span>
+    <span class="form-label" *ngIf="caseField.label">{{caseField | ccdFieldLabel}}</span>    
   </label>
+  <span class="form-hint" *ngIf="caseField.hint_text">{{caseField.hint_text}}</span>
+  <span class="error-message" *ngIf="emailControl.errors && emailControl.touched">{{emailControl.errors | ccdFirstError:caseField.label}}</span>
 
-  <input class="form-control" [id]="id()" type="email" [formControl]="emailControl">
+  <input class="form-control" [ngClass]="{'govuk-input--error': emailControl.errors && emailControl.touched}" 
+  [ngStyle]="{'border-width':emailControl.errors && emailControl.touched ? '2px':null}" [id]="id()" type="email" [formControl]="emailControl">
```

Comment:
Same as above

Reply:
As discussed removed the [ngStyle] attribute